dapps/src/tests/fetch.rs

```diff
@@ -60,7 +60,7 @@ fn should_return_503_when_syncing_but_should_make_the_calls() {
 
 	// then
 	response.assert_status("HTTP/1.1 503 Service Unavailable");
-	assert_eq!(registrar.calls.lock().len(), 4);
+	assert_eq!(registrar.calls.lock().len(), 2);
```

Comment:
is it wrong now? "the calls" is pretty ambiguous: it's definitely making some calls, although not as many as before.